CHANGELOG.md

```diff
@@ -373,10 +373,10 @@ conditions of `Napi::AsyncProgressWorker`.
 #### API
 
 - Introduced `include_dir` for use with **gyp** in a scalar context.
-- Added `Napi::Addon` to help handle the loading of a native add-on into 
+- Added `Napi::Addon` to help handle the loading of a native add-on into
 multiple threads and or multiple times in the same thread.
 - Concentrate callbacks provided to core N-API.
-- Make sure wrapcallback is used.
+- Make sure wrap callback is used.
```

Comment:
I think this refers to a `wrapcallback` property or somesuch but I'm not sure. @nodejs/node-api 

```suggestion
- Make sure `wrapcallback` is used.
```